### PR TITLE
Html Upgrade Tweaks

### PR DESCRIPTION
**Pull request by stephencelis**
Removes the underscores, unwraps a few arrays, and cleans up the about page.

### CODE REVIEW

File: Sources/FunctionalCss/GridHelpersUpgrade.swift [hunk 1]
```diff
@@ -3,33 +3,29 @@ import HtmlUpgrade
 
 // TODO: extract to grid helpers in design systems?
 extension Node {
-  public static func _gridRow(_ attribs: [Attribute<Tag.Div>], _ content: [Node]) -> Node {
-    return .div(attributes: _addClasses([Class.grid.row], to: attribs), .fragment(content))
+  public static func gridRow(attributes: [Attribute<Tag.Div>] = [], _ content: Node...) -> Node {
```
Comment (stephencelis) [inline Sources/FunctionalCss/GridHelpersUpgrade.swift]: Updated the function to match the library a bit more by removing the overload method and give attributes a default.

File: Sources/FunctionalCss/GridHelpersUpgrade.swift [hunk 2]
```diff
@@ -3,33 +3,22 @@ import HtmlUpgrade
...
+    return .div(attributes: _addClasses([Class.grid.row], to: attributes), .fragment(content))
   }
 
-  // TODO: extract to grid helpers in design systems?
-  public static func _gridRow(_ content: [Node]) -> Node {
-    return ._gridRow([], content)
-  }
-
-  // TODO: extract to grid helpers in design systems?
-  public static func _gridColumn(sizes: [Breakpoint: Int]) -> ([Node]) -> Node {
```
Comment (stephencelis) [inline Sources/FunctionalCss/GridHelpersUpgrade.swift]: Removed this curried overload for now...assuming we don't want it.

File: Sources/PointFree/PageLayout.swift [hunk 3]
```diff
@@ -107,13 +107,13 @@ struct SimplePageLayoutData<A> {
 }
 
 func _respond<A, B>(
-  view: @escaping (B) -> [HtmlUpgrade.Node],
+  view: @escaping (B) -> HtmlUpgrade.Node,
```
Comment (stephencelis) [inline Sources/PointFree/PageLayout.swift]: Now that `Node` has `fragment` I think we should rarely (if ever) work with `[Node]`.

File: Sources/Views/About.swift [hunk 4]
```diff
@@ -29,38 +29,37 @@ public struct Host {
...
-    ._gridRow([
-      ._gridColumn(sizes: [.mobile: 12, .desktop: 7], [
-        .div(
-          attributes: [.class([Class.padding([.mobile: [.all: 3], .desktop: [.all: 4]])])],
-          .fragment(aboutSectionView + openSourceSection)
-        )
-        ]),
+public func aboutView(hosts: [Host]) -> Node {
+  return .gridRow(
+    .gridColumn(
```
Comment (stephencelis) [inline Sources/Views/About.swift]: Can flatten a lot of the array stuff now and work with `Node...` now.

File: Sources/Views/About.swift [hunk 5]
```diff
@@ -29,38 +29,37 @@ public struct Host {
...
+        ],
+        hostsView(hosts: hosts)
+      )
+    )
+  )
 }
 
 private func hostsView(hosts: [Host]) -> Node {
-  return .fragment([
+  return [
```
Comment (stephencelis) [inline Sources/Views/About.swift]: `Node` is `ExpressibleByArrayLiteral`.

File: Sources/Views/About.swift [hunk 6]
```diff
@@ -71,77 +70,79 @@ private func hostsView(hosts: [Host]) -> Node {
...
+         "open source"),
       " the ",
       .a(attributes: [.class([Class.pf.colors.link.green, Class.type.underline]), .href("https://www.kickstarter.com")], "Kickstarter"),
       " mobile apps."
-      ])
-    ]
-    + hosts.flatMap(hostView)
-  )
+    ),
+    .fragment(hosts.map(hostView))
```
Comment (stephencelis) [inline Sources/Views/About.swift]: I think generally we only need to use `fragment` directly is during `map`.

File: Sources/Views/About.swift [hunk 7]
```diff
@@ -29,38 +29,37 @@ public struct Host {
...
-        )
-        ]),
+public func aboutView(hosts: [Host]) -> Node {
+  return .gridRow(
+    .gridColumn(
+      sizes: [.mobile: 12, .desktop: 7],
+      .div(
+        attributes: [.class([Class.padding([.mobile: [.all: 3], .desktop: [.all: 4]])])],
+        aboutSectionView,
+        openSourceSection
```
Comment (stephencelis) [inline Sources/Views/About.swift]: When both of these return `Node` instead they can be inlined, and no need to `.fragment(a + b)`.

File: Sources/FunctionalCss/GridHelpersUpgrade.swift (hunk 2 above)
Comment (mbrandonw) [reply Sources/FunctionalCss/GridHelpersUpgrade.swift]: yeahhhh there are maybe a few uses of it but we can just remove as we come across them

File: Sources/Views/About.swift (hunk 7 above)
Comment (mbrandonw) [reply Sources/Views/About.swift]: amazing

File: Sources/FunctionalCss/GridHelpersUpgrade.swift (hunk 2 above)
Comment (stephencelis) [reply Sources/FunctionalCss/GridHelpersUpgrade.swift]: ha let's revisit it when we need it.

**Pull request merged**